### PR TITLE
feat: upload with chunk

### PR DESCRIPTION
**Pull request by azam-ismail**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@
 %dev.quarkus.http.port=9000
 %dev.quarkus.http.limits.max-body-size=122880k
 %dev.quarkus.http.cors.origins=http://localhost:4200,http://localhost:9000,http://localhost:3000
-%dev.quarkus.mongodb.connection-string=mongodb://aemodev:yAyLMOvPmcvWfLwAfZ3LGoPDLx997WkV1ujGPkTFghWZzEhyaJ6hZyTYjYd9Qda20kn8XCFGjUZ9Tvdry1pQag==@aemodev.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@aemodev@
+%dev.quarkus.mongodb.connection-string=mongodb://root:example@localhost:27017
```
Comment (kim-energyweb) [inline src/main/resources/application.properties:8]: Wrong value?

**Review by kim-energyweb — CHANGES_REQUESTED** [review]
Update local value

File: src/main/resources/application.properties (hunk 1 above)
Comment (azam-ismail) [reply src/main/resources/application.properties:8]: that for dev local

**Review by kim-energyweb — APPROVED** [review]
LGTM

**Pull request merged**